### PR TITLE
Adds a very basic ninja antag as an example of how to add a new role.

### PR DESCRIPTION
**Pull request by MadmanMartian**
very simple, no metal gear rising/warframe/ninja gaiden stuff ~~yet~~

### CODE REVIEW

**Comment by gbasood** [comment]
oh god, do you realize what you've done here

**Comment by MadmanMartian** [comment]
I'm not seeing any glaring issues, but then again it is 2:30AM. What's wrong?

**Comment by Hoshi-chan** [comment]
But ninjas aren't honorable. Ninjas fight dirty. They're spies and assassins, and they use whatever's at hand to get the job done.

**Comment by MadmanMartian** [comment]
A fair point, but why be called a ninja if you're going to sit in the atmospheric hardsuit, plasmaflood, and shoot everyone with the captains laser?

**Comment by Hoshi-chan** [comment]
I dunno, why _would_ you call that role a ninja? A ninja would basically just be a tator unless it was a cool cyberninja like we used to have, which had better, cooler stuff than hardsuits and plasma floods. I also don't think the old ninjas could remove their suits, so there's that.

**Comment by MadmanMartian** [comment]
And this isn't old ninja, it's a quick example of how to make a new role for role datums

**Comment by Hoshi-chan** [comment]
Yeah, and I was just saying. Why are you concerned with the gameplay implications of whether or not he's honorable if it's just an example?

**Comment by MadmanMartian** [comment]
Because there's little doubt it'll get spawned in at some point, so might as well add some variety to it

**Comment by Hoshi-chan** [comment]
I have no ideas that wouldn't require new code/features. We don't even have throwing stars, do we? It seems like something we'd have, but maybe not.

Maybe consider giving ninja the clumsy gene instead, and doesn't the strong gene make thrown items do more damage? That's at least more thematically consistent than honorable, I think. I don't really have anything better, I guess. I don't know how to step away from this gracefully, so I'm just going to stop posting now.

**Comment by Blithering** [comment]
> yet

bust

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Isn't not being able to remove their suit some form of "honor"

**Comment by Kurfursten** [comment]
>Ninja
>Lives with honor
>Wears a kimono and carries a katana

I just know there's an order of the stick comic somewhere about the difference between ninjas and samurai

**Comment by jknpj** [comment]
lacking fingerless gloves

**Comment by spoonylove** [comment]
10o soon. For the Lotus I will cleanse this station of clowns.

**Comment by DeityLink** [comment]
You'll have to add example dynamic rulesets to this PR for the guide to be truly complete.

I can take care of it if you want.

File: code/datums/gamemode/misc_gamemode_procs.dm [hunk 1]
```diff
@@ -270,3 +270,31 @@ proc/name_wizard(mob/living/carbon/human/wizard_mob)
...
+	H.equip_to_slot_or_del(new /obj/item/clothing/suit/kimono/ronin, slot_wear_suit)
+	H.equip_to_slot_or_del(new /obj/item/clothing/under/color/black, slot_w_uniform)
+	H.equip_to_slot_or_del(new /obj/item/weapon/storage/belt/skull, slot_belt)
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal, slot_shoes)
+	H.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/messenger/black, slot_back)
+	H.equip_to_slot_or_del(new /obj/item/weapon/storage/box/syndie_kit/smokebombs, slot_in_backpack)
+
+
+/proc/name_ninja(var/mob/living/carbon/human/H)
+	//Allows the wizard to choose a custom name or go with a random one. Spawn 0 so it does not lag the round starting.
```
Comment (Kurfursten) [inline code/datums/gamemode/misc_gamemode_procs.dm:288]: >Wizard

**Pull request closed without merging**